contents/sprint/2025/09-hsinchu.mdx

```diff
@@ -0,0 +1,112 @@
+us](#contact-us) if you have any questions.
+
+Scisprint, hosted by the sciwork community, is a monthly coding sprint.
+It would like to facilitate discussions and exchanges among people in
+the fields of science, numerical computation, and engineering.
+Participants, regardless of experience level, can gain valuable
+development insights in this event.
+
+We would like to provide a supportive and friendly environment for all
+attendees to support more developers to join in the open-source
+communities.
+
+## Hacking Session
+
+It aims to encourage collaboration and interaction among developers
+through project participation. The projects cover various fields,
+including but not limited to science, numerical computation, and
+engineering. You are also encouraged to share your own projects in
+scisprint. Refer to [project list](#project-list) below for more
+details.
+
+## Project List
+
+### modmesh
+
+- **Related Subjects:** Python, C++, PDE
+- **Project Link:** [GitHub](https://github.com/solvcon/modmesh)
+- **Project Contact:** Yung-Yu Chen (discord: \@yyc#7718)
+
+modmesh seamlessly mixes C++ and Python through pybind11, allowing you
+to leverage the strengths of both programming languages for efficient
+PDE solving. We use Qt and Python to visualize the computation results
+to give you a better understanding of your PDE solution. modmesh also
+supports mesh visualization, currently in the Gmsh mesh file format. We
+have recently made efforts to improve the modmesh UI/UX.
+
+The design allows it to run on Windows, Linux, and MacOS. Everyone can
+use or contribute to modmesh.
+
+### AItlas
+
+- **Project Link:** [GitHub](https://github.com/Droidtown/AItlas)
+- **Project Contact:** PeterWolf (discord: \@PeterWolf#1422),
+  [Discord](https://discord.com/invite/tYq4qUY4)
+
+A knowledge graph should formally represent relations between/among
+entities and provide insights for semantic computation in natural
+language understanding (NLU). The representation is usually like a net
+2D or atlas in 3D and higher dimensions. In recent years, this is
+usually called RAG. We believe a knowledge graph (thus KG) can do more
+for AI, thus named our project AI+Atlas = AItlas.
+
+
+## Sign Up
+
+Please register at
+[kktix](https://sciwork.kktix.cc/events/scisprint-202509-hsinchu).
+
+
+## Venue
+
+### Venue Map
+
+[前沿理論及計算研究中心 (國立清華大學第三綜合大樓 A 區 5
+樓)](https://goo.gl/maps/EH2wWtkLQ8qLWd669).
+
+[Center for Theory and Computation (Rm. P518, 3rd General Building,
+National Tsing Hua University)](https://goo.gl/maps/4i2K2XvJqw2J42pv5).
+
+<Map src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d28976.98152829823!2d120.96353258312313!3d24.79125198152699!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3468360c81cfffe3%3A0xd7d529328f01b825!2z5ZyL56uL5riF6I-v5aSn5a2456ys5LiJ57ac5ZCI5aSn5qiT!5e0!3m2!1szh-TW!2stw!4v1662888048158!5m2!1szh-TW!2stw" />
+
+### Floor Plan
+
+![NTHU_CTC_floor_plan](NTHU_CTC_floor_plan.jpg)
```

Comment:
This will cause the image URL to be incorrect.

Please rewrite it as follows: (I haven't tested this, so it might need some modification)
```
import Image from "next/image";
import floorPlan from "@/assets/sprint/2025/NTHU_CTC_floor_plan.jpg";
...
<Image src={floorPlan} alt="Floor Plan" />
...
```